### PR TITLE
Breakup `R` motion-serifed variant into multiple variants, also add them for Cyrillic Ya

### PR DESCRIPTION
**Pull request by jmcwilliams403**
`R₹ꝚꝛKkκКкЯя`
`ss15`:
image
Compared to IBM Plex Mono:
image
`ss17`:
image
Compared to Recursive Sans Mono Linear:
image
image

### CODE REVIEW

File: changes/26.3.3.md [hunk 1]
```diff
@@ -1,7 +1,16 @@
 * \[**Breaking**\] Change of variant names:
```
Comment (be5invis) [inline changes/26.3.3.md:1]: If it is breaking, then make the version number 27.0.0, and remove all lines in `variants.toml` containing `nonBreakingVariantAdditionPriority` or `nonBreakingTagForNewVariantSelector`.

Comment (be5invis) [reply changes/26.3.3.md:1]: In addition, _measure_ the impact of glyph quantity of your change.

**Review by be5invis — CHANGES_REQUESTED** [review]

**Comment by jmcwilliams403** [comment]
I can't currently change the version name of the \.md file without breaking parity with the dev branch as of 366f377

**Comment by jmcwilliams403** [comment]
Also I chose the breaking option because I already made a breaking change with #1986 and I figured it would be more effort than it's worth to turn everything so far into a non-breaking change at this point.

**Comment by jmcwilliams403** [comment]
Nevermind I just realized that 26.3.3 is already out. I'll start this commit over later without breaking anything.

**Comment by be5invis** [comment]
@jmcwilliams403 I'd like to merge this now as I decided to create version 27.0.
No need for a new PR -- I will fix everything later.

**Pull request merged**